test/pg_regress.sh

```diff
@@ -30,6 +30,9 @@ SKIPS=${SKIPS:-}
 PSQL=${PSQL:-psql}
 PSQL="${PSQL} -X" # Prevent any .psqlrc files from being executed during the tests
 
+export PSQL
```

Comment:
seems like this is an unrelated change not belonging to this PR

Reply:
these changes enable me to debug tests - removed it

Reply:
You can put it in separate PR or at least separate commit.

Reply:
I will - and there I'll explain how this could be usefull